src/til/ut_til/EnumSetTests.cpp

```diff
@@ -53,39 +53,39 @@ class EnumSetTests
 
         Log::Comment(L"Start with no bits set");
         til::enumset<Flags> flags;
-        VERIFY_ARE_EQUAL(0b00000u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b00000u, flags.bits());
 
         Log::Comment(L"Set bit 2 to true");
         flags.set(Flags::Two);

```

Comment:
The point of this test case was originally to validate the single position `set` method with a `val` of true. With the new method naming this is going to use the multiple position method instead. To get the intended coverage, I'd suggest changing at least one of these `set` calls to:
```
flags.set(Flags::Two, true);
```